src/components/Annotation/TagList.css

```diff
@@ -9,6 +9,10 @@
   padding: 0 0 10px 0; 
 }
 
+.annotation .taglist button {
+  height: auto;
```

Comment:
I noticed that tags appeared to high in the annotation editor. This seems to have been caused by a blanket `height: 38px` rule that got introduced for the `button` element at some point. This fix patches the problem.